src/sympc/tensor/mpc_tensor.py

```diff
@@ -736,6 +753,34 @@ def _get_shape(
         res = op(x, y, **kwargs_)
         return res.shape
 
+    def truncation(
```

Comment:
Splitting the truncation would be great :+1:

Reply:
You mean I should split? or it's good that I split it.